docs/userstylesyml.md

```diff
@@ -0,0 +1,149 @@
+     color: <color>
+     readme:
+       app-link: <url>
+       current-maintainers: [*<github-username>]
+   ```
+
+   `All Fields:`
+
+   These extra `usage` & `faq` keys will ensure that you can add important
+   information about the port to the README.
+
+   The `icon` key is best explained the schema specification itself.
+
+   The `past-maintainers` key is a list of people who have maintained the port
+   in the past. We encourage all maintainers to add/remove themselves from this
+   list as they see fit.
+
+   Remember that these 5 fields are **optional**.
+
+   ```yaml
+   <port>:
+     name: <name>
+     categories: [<primary-category>, <optional-secondary-category>, <optional-third-category>]
+     color: <color>
+     icon: <icon>                                    # OPTIONAL
+     readme:
+       app-link: "<url>"
+       usage: |+                                     # OPTIONAL
+         > [!NOTE]
+         > <your notes>
+       faq:                                          # OPTIONAL
+         - question: <question>
+           answer: <answer>
+       current-maintainers: [*<github-username>]
+       past-maintainers: [*<github-username>]        # OPTIONAL
+   ```
+
+### Adding yourself as a maintainer
+
+To add yourself as a maintainer, you need to add your GitHub username to the
+`maintainers` array in the `userstyles.yml` file. This file is located in the
+`scripts` directory. The `maintainers` array, is an array of objects, each
+object should contain a `name` and a `url` key. And optionally a `name` key,
+which is your preferred name.
+
+```yaml
+maintainers:
+  - &your-username
+    name: Your Preferred Name
+    url: https://github.com/your-username
+```
+
+Then you need to add your username to the `current-maintainers` array in for the
+userstyle you want to maintain, in this case `youtube`.
+
+```yaml
+userstyles:
+  youtube:
+    name: YouTube
+    categories: [entertainment, social_networking, photo_and_video]
+    icon: youtube
+    color: red
+    readme:
+      app-link: "https://youtube.com"
+      current-maintainers: [*isabelroses *your-username]
+      past-maintainers: [*elkrien]
+```
+
+### Removing yourself as a maintainer
+
+To remove yourself as a maintainer, you need to remove your username from the
+`current-maintainers` array for the userstyles you wish to no longer maintain,
+and add yourself to the `past-maintainers` array. In the example of youtube
+below, `elkrien` has been removed from the `current-maintainers` array and added
+to the `past-maintainers` array.
+
+```yaml
+userstyles:
+  youtube:
+    name: YouTube
+    categories: [entertainment, social_networking, photo_and_video]
+    icon: youtube
+    color: red
+    readme:
+      app-link: "https://youtube.com"
+      current-maintainers: [*isabelroses]
+      past-maintainers: [*elkrien]
+```
+
+If you prefer not to be listed as a past maintainer, you can remove yourself for
+maintainers array entirely.
```

Comment:
Maybe just one more sentence saying that they'll be removed from the `userstyle-maintainers` GitHub team?